src/garage/torch/algos/vpg.py

```diff
@@ -79,6 +81,8 @@ def __init__(
         self._use_softplus_entropy = use_softplus_entropy
         self._stop_entropy_gradient = stop_entropy_gradient
         self._entropy_method = entropy_method
+        self.minibatch_size = minibatch_size
```

Comment:
all members variables should be `_private`

Reply:
fixed